docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md

```diff
@@ -37,58 +37,42 @@ authentication on the server computer.
-Name                       OwnerNode                           State 
-----                       ---------                           ----- 
-Virtual Machine1           node2                              Online
+```powershell
+Move-ClusterVirtualMachineRole -Name "Virtual Machine1" -Node node2
 ```
 
 This example performs a live migration of the clustered virtual machine named Virtual Machine1 to
-the node named node2. The Windows PowerShell prompt doesn't return until the action is complete.
+the node named `node2`. The Windows PowerShell prompt doesn't return until the action is complete.
 
 ### Example 2
 
-```
-PS C:\> Get-ClusterGroup -Name "Virtual Machine1" | Move-ClusterVirtualMachineRole -Node node2 -Wait 0
-Name                       OwnerNode                           State 
-----                       ---------                           ----- 
-Virtual Machine1           node2                              Online
+```powershell
+Get-ClusterGroup -Name "Virtual Machine1" | Move-ClusterVirtualMachineRole -Node node2 -Wait 0
 ```
 
-This example performs a live migration of clustered virtual machine named Virtual Machine1 to the
-node named node2. The Windows PowerShell® prompt returns as soon as the action has been initiated.
+This example performs a live migration of clustered virtual machine named `Virtual Machine1` to the
+node named `node2`. The Windows PowerShell prompt returns as soon as the action has been initiated.
 
 ### Example 3
 
-```
-PS C:\> Move-ClusterVirtualMachineRole -Name "Virtual Machine1" -Cancel
-Name                       OwnerNode                           State 
-----                       ---------                           ----- 
-Virtual Machine1           node1                              Online
+```powershell
+Move-ClusterVirtualMachineRole -Name "Virtual Machine1" -Cancel
 ```
 
-This example cancels the live migration in progress for the clustered virtual machine named Virtual
-Machine1.
+This example cancels the live migration in progress for the clustered virtual machine named `Virtual
+Machine1`.
 
 ### Example 4
 
-```
-PS C:\> $groups = Get-ClusterNode -Name node1 | Get-ClusterGroup | Where-Object -FilterScript {$_ | Get-ClusterResource | Where-Object -FilterScript {$_.ResourceType -Like "Virtual Machine"}}
-
-
-PS C:\> ForEach-Object -InputObject $groups -Process { $_ | Move-ClusterVirtualMachineRole -Node node2 }
-Name                       OwnerNode                           State 
-----                       ---------                           ----- 
-Virtual Machine1           node2                              Online 
-Virtual Machine2           node2                              Online 
-Virtual Machine3           node2                              Online
+```powershell
+$groups = Get-ClusterNode -Name node1 | Get-ClusterGroup | Where-Object -FilterScript {$_ | Get-ClusterResource | Where-Object -FilterScript {$_.ResourceType -Like "Virtual Machine"}}
```

Comment:
Hey @michaeltlombardi - I'm having a bad brain moment! Can you think of a nicer way to write this without rolling over the line?

Thanks
Robin

Reply:
@robinharwood Happy with the PR aside from this last comment to resolve.